index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <header>
+    <body>
+      <main>
+        <section>
+          <article>
+            <div>
+              <p>Order Id: 123098097124</p>
+              <p>Placed On: 11/12/2021</p>
+            </div>
+            <div>
+              <p>View Details</p>
+            </div>
+            <div>
+              <p>-----------------------------------------------------------------------------------</p>
```

Comment:
I want to see what these look like when I start doing the CSS editing, and if I have to make these lines a little bit shorter.